### PR TITLE
Pod Node Selector Config

### LINKED ISSUE #2884: Add admission plugins configuration
Add new input (the same as for the label) if PodNodeSelector is enabled during cluster creation or update.
The cluster structure is extended for the new field: `"podNodeSelectorAdmissionPluginConfig":{"clusterDefaultNodeSelector":"env=development"}` in cluster spec. The user should be able to add /edit this map where key is a namespace, value is a label selector.

### PR DESCRIPTION
**Pull request by kgroschoff**
**What this PR does / why we need it**:
Added option to specify a Pod Node Selector Configuration during cluster creation or cluster edit. It will only be visible, when Pod Node Selector is enabled as Admission Plugin.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2884

**Special notes for your reviewer**:
podNodeSelectorConfig1
podNodeSelectorConfig2
podNodeSelectorConfig3

**Release note**:

```release-note
Add option to specify Pod Node Selector Configuration
```

### CODE REVIEW

**Comment by kgroschoff** [comment]
@maciaszczykm @floreks PTAL
also maybe @zreigz it would be good if you could have a brief look, if this is looking as expected

**Comment by zreigz** [comment]
I will take a look, thanks

**Comment by zreigz** [comment]
looks and works fine :+1:

File: src/app/shared/services/cluster.service.ts [hunk 1]
```diff
@@ -93,6 +93,11 @@ export class ClusterService {
     this._cluster.labels = labels;
   }
 
+  set podNodeSelectorAdmissionPluginConfig(config: object) {
+    delete this._cluster.spec.podNodeSelectorAdmissionPluginConfig;
```
Comment (maciaszczykm) [inline src/app/shared/services/cluster.service.ts:97]: Is this line required? In the next one we are overwriting the same property.

Comment (kgroschoff) [reply src/app/shared/services/cluster.service.ts]: you're right. removed it.

**Pull request merged**